### PR TITLE
Update ghoul.xml

### PR DESCRIPTION
**Pull request by AngelOfDeaths**
-Orginal Armor and Speed From (tibia Cyclopedia).

### CODE REVIEW

**Comment by AngelOfDeaths** [comment]
there is elemental, but i doesnt know how this work so if someone know please explain thanks.

	<elements>
		<element physicalPercent="100"/>		
		<element energyPercent="70"/>			
		<element earthPercent="80"/>			
		<element holyPercent="125"/>			
		<element firePercent="100"/>			
		<element icePercent="90"/>				
		<element deathPercent="0"/>				

	</elements>

File: data/monster/Undead_Humanoids/ghoul.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<monster name="Ghoul" nameDescription="a ghoul" race="blood" experience="85" speed="170" manacost="450">
+<monster name="Ghoul" nameDescription="a ghoul" race="blood" experience="85" speed="72" manacost="450">
```
Comment (TheSumm) [inline data/monster/Undead_Humanoids/ghoul.xml]: You have to double the speed if you take speed values from real tibia.

**Comment by DSpeichert** [comment]
Who knows what's the ratio between values published on e.g. tibia.wikia.com and what we have?

**Comment by DSpeichert** [comment]
Once we know the source and proper ratio/multiplayer for TFS we can re-open this.

**Comment by ranisalt** [comment]
What about an automated tool to update every once in a while?

**Comment by DSpeichert** [comment]
Sure, but where from and how?

**Comment by ranisalt** [comment]
For basic data, such as hitpoints, immunities and other general properties are fetchable from tibia.wikia.com and tibia-stats.com, the latter even has loot chances.

**Pull request closed without merging**